crates/nu-command/src/input_handler.rs

```diff
@@ -86,3 +86,33 @@ where
         }
     }
 }
+
+// A macro for handling the grapheme_cluster related flags.
+#[macro_export]
+macro_rules! grapheme_flags {
```

Comment:
Hi, what's the reason to define `grapheme_flags` as a macro?  I think it can be handled by rust function.

And since is restricted in str command usage, it'd be better defined inside `strings` module(put it to `strings/mod.rs` is fine), we don't need to put it into `input_handler.rs` file